update/sync/syncHeaders.go

```diff
@@ -203,10 +203,12 @@ func (h *headersToSync) SyncUnFinishedMetaHeaders(epoch uint32) error {
 
 // SyncEpochStartMetaHeader syncs and validates an epoch start metaHeader
 func (h *headersToSync) syncEpochStartMetaHeader(epoch uint32, waitTime time.Duration) error {
+	log.Debug("headersToSync.syncEpochStartMetaHeader started") //TODO remove this
```

Comment:
we might remove all TODOs from this file. It helps the debugging process and should not pollute the log file